device-types/Huawei/NE8000-F1A-8H20Q.yaml

```diff
@@ -0,0 +1,134 @@
+---
+manufacturer: Huawei
+model: NE8000 F1A-8H20Q
+slug: huawei-ne8000-f1a-8h20q
+part_number: 02353AES
+u_height: 1.0
+is_full_depth: true
+airflow: front-to-rear
+comments: '[NetEngine 8000 F1A Universal Service Router](https://e.huawei.com/en/material/networking/service-router/b332209e877b43ccbf162820cd96c756)'
+power-ports:
+  - name: PSU 1
+    type: iec-60320-c14
+    maximum_draw: 600
+    allocated_draw: 325
+  - name: PSU 2
+    type: iec-60320-c14
+    maximum_draw: 600
+    allocated_draw: 325
```

Comment:
Please correct me if I am wrong, but it looks like the PSUs are modular. If that is true, these should be defined as `module-bays` and the associated PSUs should be created in the module-types folder